phpBB/language/en/composer.json

```diff
@@ -0,0 +1,27 @@
+{
+	"name": "en",
```

Comment:
True. But you'll never know the vendor without looking it up.

Also, we need to change phpBB